.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,38 +5,38 @@ SPDX-License-Identifier: Apache-2.0
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: ''
+title: [Component name] Problem to solve, maybe how
```

Comment:
Change to: `title: [Component name] Undesired behavior observed, maybe the culprit`